applications/StructuralMechanicsApplication/custom_strategies/custom_strategies/eigensolver_strategy.hpp

```diff
@@ -583,6 +587,69 @@ class EigensolverStrategy
     ///@name Private Operations
     ///@{
 
+    /// Recover the solution related to the slave dofs in case of master-slave constraints.
+    /**
+     *  This function is an adaptation of the implementation in ConstraintUtilities,
```

Comment:
```suggestion
     * @brief Recover the solution related to the slave dofs in case of master-slave constraints.
     * @details This function is an adaptation of the implementation in ConstraintUtilities,
```